notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb

```diff
@@ -0,0 +1,406 @@
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Imports modules required to run"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "import cv2\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from openvino.inference_engine import IECore\n",
+    "from shutil import copy\n",
+    "from os import path, makedirs, listdir"
+   ],
+   "outputs": [],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Settings"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "ie = IECore()\n",
+    "\n",
+    "model_folder = \"model\"\n",
+    "download_folder = \"output\"\n",
+    "\n",
+    "precision = \"FP16\"\n",
+    "detection_model_name = \"horizontal-text-detection-0001\"\n",
+    "recognition_model_name = \"text-recognition-resnet-fc\"\n",
+    "model_extensions = (\"bin\", \"xml\")"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Download models and convert public model\n",
+    "\n",
+    "If it is your first run models will download and convert here. It might take up to ten minutes. "
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "makedirs(model_folder, exist_ok=True)\n",
+    "makedirs(download_folder, exist_ok=True)\n",
+    "\n",
+    "# Check if models are already downloaded in download directory\n",
+    "try:\n",
+    "    for model_name, folder_name in ((detection_model_name, f'intel/{detection_model_name}'), (recognition_model_name, f'public/{recognition_model_name}')):\n",
+    "        for extension in model_extensions:\n",
+    "            if not path.isfile(f'{download_folder}/{folder_name}/{precision}/{model_name}.{extension}'):\n",
+    "                raise FileNotFoundError\n",
+    "except FileNotFoundError:\n",
+    "    download_command = f\"omz_downloader --name {detection_model_name},{recognition_model_name} --output_dir {download_folder} --precision {precision}\"\n",
+    "    convert_command = f\"omz_converter --name {recognition_model_name} --precisions {precision} --download_dir {download_folder} --output_dir {download_folder}\"\n",
+    "    # Run commands, first download model than convert it to inferable \n",
+    "    !! $download_command\n",
+    "    print('Models downloded')\n",
+    "    # Models are downloaded straight to output folder, we will keep all not used files outside of models directory\n",
+    "    !! $convert_command\n",
```

Comment:
```suggestion
    "    ! $convert_command\n",
```

Reply:
I want to show the convert output, so please change (and continue to hide the model downloader output)